maistra/run-ci.sh

```diff
@@ -15,26 +15,32 @@ export ARCH
 export BUILD_SCM_REVISION="Maistra PR #${PULL_NUMBER:-undefined}"
 export BUILD_SCM_STATUS="SHA=${PULL_PULL_SHA:-undefined}"
 
+COMMON_FLAGS="\
+    --incompatible_linkopts_to_linklibs \
+    --local_ram_resources=12288 \
+    --local_cpu_resources=6 \
+    --jobs=3 \
```

Comment:
why the reduction to 3 jobs?

Reply:
we had issues with 4 jobs in 2.2, so, let's reduce everywhere on demand